files/ohpc_install.yml

```diff
@@ -98,12 +106,28 @@
                                 enable_nfs_ohpc=True
                                 enable_nfs_home=True
                                 enable_helloworld=True
+                                enable_socket_direct=False
```

Comment:
I would set socket direct support tied to Mofed vs. OSS, as this is the *only* reason we support it.

Reply:
Yes that's what we're doing here, this is false by default.

Reply:
I meant, leave it as false by default, but move the enablement into the if/else block below (if ib_install == mofed).